contrib/scripts/golangci-lint.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+GOLANGCI_LINT_BIN="${GOLANGCI_LINT_BIN:-golangci-lint}"
+GOLANGCI_LINT_ARGS="${GOLANGCI_LINT_ARGS:-}"
+GOLANGCI_LINT_MODULE="${GOLANGCI_LINT_MODULE:-golangci-lint-cilium}"
+
+
+# If the golangci-lint custom module binary does NOT exist,
+# build the custom golangci-lint module using `golangci-lint custom`.
+if  [ ! -x "${GOLANGCI_LINT_MODULE}" ]; then
+	echo "'${GOLANGCI_LINT_MODULE}' does not exist"
+	echo "'${GOLANGCI_LINT_MODULE}' build"
+	
+	"${GOLANGCI_LINT_BIN}" custom
+else
+	echo "'${GOLANGCI_LINT_MODULE}' exist"
```

Comment:
nit: Ideally `make --quiet V=0 <target>` is very quiet. Also any messages printed by this script should preferably not be part of stdout for the results of the testing. 

I'd suggest either deleting all the echo lines, or if you feel it's better, minimize to just one line which also emits to stderr:

```suggestion
	echo >&2 "'${GOLANGCI_LINT_MODULE}' build"
	
	"${GOLANGCI_LINT_BIN}" custom
```

Reply:
Okay, I’ll remove the echo